src/EditorFeatures/VisualBasicTest/QuickInfo/SemanticQuickInfoSourceTests.vb

```diff
@@ -2032,5 +2032,32 @@ End Interface
             TestFromXml(markup, MainDescription("Interface IFoo"), Documentation("summary for interface IFoo"))
         End Sub
 
+        <Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)>
+        <WorkItem(4868, "https://github.com/dotnet/roslyn/issues/4868")>
+        Public Sub QuickInfoExceptions()
+            Test("
+Imports System
+Namespace MyNs
+    Class MyException1
+        Inherits Exception
+    End Class
+    Class MyException2
+        Inherits Exception
+    End Class
+    Class TestClass
+        ''' <exception cref=""MyException1""></exception>
+        ''' <exception cref=""T:MyNs.MyException2""></exception>
+        ''' <exception cref=""System.Int32""></exception>
+        ''' <exception cref=""Double""></exception>
+        ''' <exception cref=""Not_A_Class_And_Therefore_Not_Listed""></exception>
+        Sub M()
+            M$$()
+        End Sub
+    End Class
+End Namespace
+",
+                Exceptions($"{vbCrLf}{WorkspacesResources.Exceptions}{vbCrLf}  MyException1{vbCrLf}  MyException2{vbCrLf}  Double{vbCrLf}  Integer"))
```

Comment:
Thanks for the pointer.  I've submitted a corresponding internal PR to test this.